doc/REST-interface.md

```diff
@@ -47,6 +47,11 @@ The HTTP request and response are both handled entirely in-memory.
 
 With the /notxdetails/ option JSON response will only contain the transaction hash instead of the complete transaction details. The option only affects the JSON response.
 
+- `GET /rest/blockpart/<BLOCK-HASH>.<bin|hex>?offset=X&size=Y`
```

Comment:
if we're writing `<BLOCK-HASH>` and not just `BLOCK-HASH`, we should so the same for `X` and Y` (which can also be reworded for consistency):
```suggestion
- `GET /rest/blockpart/<BLOCK-HASH>.<bin|hex>?offset=<OFFSET>&size=<SIZE>`
- ```